translations/messages+intl-icu.de.yml

```diff
@@ -3360,6 +3360,7 @@ statement.metadata: "Metadaten"
 error.statement.missing.segment.drafts: "Bitte teilen Sie die Stellungnahme zuerst in Abschnitte auf. Erst dann können Sie das Aufteilen abschließen."
 statement.modal.step.write: "Verfassen Sie Ihre Stellungnahme und geben Sie Metadaten wie Verortung oder Dokumentenbezug an."
 statement.modal.step.personal.data: "Wählen Sie zwischen anonymer und namentlicher Einreichung und geben ggf. Ihre Kontaktdaten an."
+statement.modal.step.privacy.policy: "Bitte beachten Sie die geltende <a class=\"underline color-alt-contrast\" href=\"/datenschutz\" target=\"_blank\" rel=\"noopener noreferrer\">Datenschutzerklärung</a>."
```

Comment:
The `color-alt-contrast` class can be removed (it is not applied). I think we should adjust the styling though as the link color doesn't have enough contrast with the background color of the info box.

Reply:
```suggestion
statement.modal.step.privacy.policy: "Bitte beachten Sie die geltende <a class=\"underline\" href=\"/datenschutz\" target=\"_blank\" rel=\"noopener noreferrer\">Datenschutzerklärung</a>."
```